### DISCUSSION TITLE
Heath Death of the universe - gravity keeps the universe out of thermal equilibrium

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#120] [Cosmology Models]
There is no heat death in a static universe.
From wikipedia [https://en.wikipedia.org/wiki/Heat_death_of_the_universe#Opposing_views](https://en.wikipedia.org/wiki/Heat_death_of_the_universe#Opposing_views)

> bound systems have negative specific heat—that is, the velocities of their components increase when energy is removed. ... Such a system does not evolve toward a homogeneous equilibrium state. Instead it becomes increasingly structured and heterogeneous as it fragments into subsystems.

> It should be expected that an isolated system fragmented into subsystems does not necessarily come to thermodynamic equilibrium and remain in non-equilibrium steady state. Entropy will be transmitted from one subsystem to another, but its production will be zero, which does not contradict the second law of thermodynamics.

Accordingly, claims that *a static cosmology is inconsistent with the second law of thermodynamics* are incorrect.

@MJoseph315, Arthur Milne and Lee Smolin are two of the many cosmologists who have argued against the "heath death".

**Comment by ExpEarth** [comment]
A longer quote from that Wikipedia article:

> A 2010 analysis of entropy states, "The entropy of a general gravitational field is still not known", and "gravitational entropy is difficult to quantify." The analysis considers several possible assumptions that would be needed for estimates and suggests that the [observable universe]( has more entropy than previously thought. This is because the analysis concludes that supermassive black holes are the largest contributor.[[31]](https://en.wikipedia.org/wiki/Heat_death_of_the_universe#cite_note-31) [Lee Smolin](https://en.wikipedia.org/wiki/Lee_Smolin) goes further: "It has long been known that gravity is important for keeping the universe out of thermal equilibrium. Gravitationally bound systems have negative specific heat—that is, the velocities of their components increase when energy is removed. ... Such a system does not evolve toward a homogeneous equilibrium state. Instead it becomes increasingly structured and heterogeneous as it fragments into subsystems." This point of view is also supported by the fact of a recent experimental discovery of a stable non-equilibrium steady state in a relatively simple closed system. It should be expected that an isolated system fragmented into subsystems does not necessarily come to thermodynamic equilibrium and remain in non-equilibrium steady state. Entropy will be transmitted from one subsystem to another, but its production will be zero, which does not contradict the second law of thermodynamics.

This is exactly how it works in my own model.  Energy is first removed from the CMB to drive gravity.  This increases the entropy of the CMB photons, but reduces entropy in the graviton filaments which make up spacetime.  The energy in those filaments is equal to the system's gravitational energy and that energy is increased and concentrated due to the input from the CMB.  Then when those filaments are in turn redshifted, energy is returned to the CMB photons, creating a blueshift in them and reducing the CMB entropy back to where it was.

**Reply by HanDeBruijn** [reply]
For what it's worth: [G. de Vaucouleurs](http://www.alternatievewiskunde.nl/cosmology/helaas.htm) section at my website, last part.

**Reply by RedshiftDrift** [reply]
@ExpEarth the CMB is ***not*** a gravitationally bound system, so removing energy does not increase entropy.

**Reply by ExpEarth** [reply]
Roger on that, Louis.  I got excited about the idea of a system gaining energy by removing energy, not recognizing it as the usual case with gravitational potential energy.  I'm not sure it's connected to my model at all!  Apologies for hasty posting.
